OpenDreamClient/Interface/DreamInterfaceManager.cs

```diff
@@ -390,26 +395,74 @@ sealed class DreamInterfaceManager : IDreamInterfaceManager {
             }
         }
 
+        public void WinClone(string controlId, string cloneId) {
+            ElementDescriptor elementDescriptor = null;
+            // window/pane
+            foreach (var windowDescriptor in InterfaceDescriptor.WindowDescriptors) {
+                if (windowDescriptor.Name == controlId) {
+                    elementDescriptor = windowDescriptor.WithName(_serializationManager, cloneId);
+                    elementDescriptor = ((WindowDescriptor)elementDescriptor).WithVisible(_serializationManager, false); // per byond spec
+                    break;
+                }
+            }
+
+            // menu
+            if (elementDescriptor == null) {
+                foreach (var menuDescriptor in InterfaceDescriptor.MenuDescriptors) {
+                    if (menuDescriptor.Name == controlId) {
+                        elementDescriptor = _serializationManager.CreateCopy(menuDescriptor);
```

Comment:
This ends up not setting the name of the new element. I tried using `WithName()`, but that ended up creating an `ElementDescriptor` instead of a `MenuDescriptor`.